src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CSIPReport.kt

```diff
@@ -0,0 +1,110 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.OneToMany
+import jakarta.persistence.SequenceGenerator
+import jakarta.persistence.Table
+import org.hibernate.Hibernate
+import org.hibernate.annotations.Generated
+import org.hibernate.annotations.JoinColumnOrFormula
+import org.hibernate.annotations.JoinColumnsOrFormulas
+import org.hibernate.annotations.JoinFormula
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.helper.EntityOpen
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "OFFENDER_CSIP_REPORTS")
+@EntityOpen
+data class CSIPReport(
+  @Id
+  @Column(name = "CSIP_ID")
+  @SequenceGenerator(name = "CSIP_ID", sequenceName = "CSIP_ID", allocationSize = 1)
+  @GeneratedValue(generator = "CSIP_ID")
+  val id: Long = 0,
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "OFFENDER_BOOK_ID")
+  val offenderBooking: OffenderBooking?,
```

Comment:
Is this really optional? Feels weird to have CSIP with no associated booking

Reply:
Well, it's actually nullable, but there are no offenders with this as null.  I'll check the db - maybe it's an oversight on the nullability of fields